src/index.js

```diff
@@ -35,144 +32,23 @@ export { default as ZoomMouseWheelTool } from './tools/ZoomMouseWheelTool.js';
-export {
-  default as stackScrollSynchronizer
-} from './synchronization/stackScrollSynchronizer.js';
-export {
-  default as stackImagePositionSynchronizer
-} from './synchronization/stackImagePositionSynchronizer.js';
-export {
-  default as stackImagePositionOffsetSynchronizer
-} from './synchronization/stackImagePositionOffsetSynchronizer.js';
-export {
-  default as stackImageIndexSynchronizer
-} from './synchronization/stackImageIndexSynchronizer.js';
-export {
-  default as panZoomSynchronizer
-} from './synchronization/panZoomSynchronizer.js';
-
-export { default as toolStyle } from './stateManagement/toolStyle.js';
-export {
-  addToolState,
-  getToolState,
-  removeToolState,
-  clearToolState,
-  setElementToolStateManager,
-  getElementToolStateManager
-} from './stateManagement/toolState.js';
-export {
-  default as toolCoordinates
-} from './stateManagement/toolCoordinates.js';
-export { default as toolColors } from './stateManagement/toolColors.js';
-export {
-  addTimeSeriesStateManager,
-  newTimeSeriesSpecificToolStateManager
-} from './stateManagement/timeSeriesSpecificStateManager.js';
-export { default as textStyle } from './stateManagement/textStyle.js';
-
-export {
-  stackSpecificStateManager,
-  newStackSpecificToolStateManager,
-  addStackStateManager
-} from './stateManagement/stackSpecificStateManager.js';
-
-export {
-  default as loadHandlerManager
-} from './stateManagement/loadHandlerManager.js';
-
-export {
-  newImageIdSpecificToolStateManager,
-  globalImageIdSpecificToolStateManager
-} from './stateManagement/imageIdSpecificStateManager.js';
-
-export {
-  newFrameOfReferenceSpecificToolStateManager,
-  globalFrameOfReferenceSpecificToolStateManager
-} from './stateManagement/frameOfReferenceStateManager.js';
-
+// ~~~~~~ STACK TOOLS  ~~~~~ //
 export { default as stackPrefetch } from './stackTools/stackPrefetch.js';
 export { default as stackRenderers } from './stackTools/stackRenderers.js';
 export { playClip, stopClip } from './stackTools/playClip.js';
 
-export {
-  default as anyHandlesOutsideImage
-} from './manipulators/anyHandlesOutsideImage.js';
-export { default as drawHandles } from './manipulators/drawHandles.js';
-export {
-  default as getHandleNearImagePoint
-} from './manipulators/getHandleNearImagePoint.js';
-export { default as handleActivator } from './manipulators/handleActivator.js';
-export { default as moveAllHandles } from './manipulators/moveAllHandles.js';
-export { default as moveHandle } from './manipulators/moveHandle.js';
-export { default as moveNewHandle } from './manipulators/moveNewHandle.js';
-export {
-  default as moveNewHandleTouch
-} from './manipulators/moveNewHandleTouch.js';
-export {
-  default as touchMoveAllHandles
-} from './manipulators/touchMoveAllHandles.js';
-export { default as touchMoveHandle } from './manipulators/touchMoveHandle.js';
+// Exposed entities for 3rd party usage/plugins.
+export { default as import } from './import.js';
```

Comment:
Should we also expose `register`? What if I want to register a single mixin? IE. I have my own codebase, and I'm not a plugin author. I just have a list of mixins I'd like to add?